webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -149,7 +149,7 @@
                             </tr>
                             <tr>
                                 <th scope="row">OS</th>
-                                <td>{{ report.os_name }}</td>
+                                <td>{{ report.os_pretty_version or report.os_name }}</td>
```

Comment:
Cool. So you've tested it. I just got nervous after the problems we had with `{{ report.InstallTime | print_date }}`